src/storage/store.js

```diff
@@ -65,6 +65,7 @@ export const SCHEMA = {
       properties: {
         displayName: { type: "string", pattern: "^[A-Za-z0-9_~ -]{3,32}$" },
         avatarId: { type: "string" },
+        pronouns: { type: "string", pattern: "^[a-z(?:/)a-z]{6,12}$" },
```

Comment:
Nit: If we don’t need the capture groups we can also do: `/^[a-zA-Z///a-zA-Z]{6,12}$/` and also match capitals (maybe we don’t want that, not sure…). Also not sure about this but the min/max length might need to be different in other languages than English so maybe we want to keep it more open to min 1 or 2 characters each and keep the max a bit higher?

Reply:
oh thank you! good catch on the other languages.